website/docs/r/bcmdataexports_export.html.markdown

```diff
@@ -10,8 +10,6 @@ description: |-
 
 Terraform resource for managing an AWS BCM Data Exports Export.
 
-~> **NOTE:** Including `line_item_resource_id` in the SQL query in `export.0.data_query.0.query_statement` results in an error, `ValidationException: The columns in the query provided are not a subset of the table COST_AND_USAGE_REPORT`. Based on [AWS Documentation](https://docs.aws.amazon.com/cur/latest/userguide/data-dictionary.html), `line_item_resource_id` should be a valid column but is currently causing errors.
```

Comment:
If this is now fixed, should we also remove the corresponding changelog entry for #37189 ?

https://github.com/hashicorp/terraform-provider-aws/pull/37189/files#diff-8b57c9c1e471066746a7107c58c2f107503d3bc54d721c463209227995c3088c